pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -61,8 +61,30 @@ const (
 
 	operationPollInterval        = 3 * time.Second
 	operationPollTimeoutDuration = 30 * time.Minute
+
+	defaultLBSourceRange = "0.0.0.0/0"
+
+	//Expected annotations for GCE
+	gceLBAllowSourceRange = "net.beta.kubernetes.io/gce-source-ranges"
```

Comment:
Yeah if we get > 1 cloudprovider to implement support for this, it's better to remove gce, but otherwise having gce in the annotation name is probably the clearest way to say it works nowhere else (if someone is just copy pasting kubectl -o output, for example)